<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lb1">
       <property name="styleSheet">
        <string notr="true">font: 57 30pt &quot;WenQuanYi Zen Hei Sharp&quot;;
</string>
       </property>
       <property name="text">
        <string>From </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbFrom">
       <property name="styleSheet">
        <string notr="true">font: 57 30pt &quot;WenQuanYi Zen Hei Sharp&quot;;
color: rgb(239, 41, 41);</string>
       </property>
       <property name="text">
        <string>XXX</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lb2">
       <property name="styleSheet">
        <string notr="true">font: 57 30pt &quot;WenQuanYi Zen Hei Sharp&quot;;
</string>
       </property>
       <property name="text">
        <string>To </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTo">
       <property name="styleSheet">
        <string notr="true">font: 57 30pt &quot;WenQuanYi Zen Hei Sharp&quot;;
color: rgb(239, 41, 41);</string>
       </property>
       <property name="text">
        <string>XXX</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 24pt &quot;WenQuanYi Zen Hei Sharp&quot;;
color: rgb(255, 0, 255);</string>
       </property>
       <property name="text">
        <string>最少换乘： </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbMinBus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;
background-color: rgba(211, 215, 207, 0.85);</string>
       </property>
       <property name="text">
        <string>'审计局站', '工学院站', '劳动局站', '交通局站', '五金市场'审计局站', '工学院站', '劳动局站', '交通局站', '五金市场'审计局站', '工学院站', '劳动局站', '交通局站', '五金市场'审计局站', '工学院站', '劳动局站', '交通局站', '五金市场</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 24pt &quot;WenQuanYi Zen Hei Sharp&quot;;
color: rgb(255, 0, 255);</string>
       </property>
       <property name="text">
        <string>最短距离： </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbMinDistance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 14pt &quot;WenQuanYi Zen Hei Sharp&quot;;
background-color: rgba(211, 215, 207, 0.85);</string>
       </property>
       <property name="text">
        <string>'审计局站', '工学院站', '劳动局站', '交通局站', '五金市场'审计局站', '工学院站', '劳动局站', '交通局站', '五金市场</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1,2">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
